bin/oref0-setup.sh

```diff
@@ -1040,39 +1011,37 @@ if prompt_yn "" N; then
-            #openaps alias add wait-for-long-silence '! bash -c "echo -n \"Listening: \"; for i in $(seq 1 200); do echo -n .; mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 45 2>/dev/null | egrep -v subg | egrep No && break; done"'
-            if [[ ${radio_locale,,} =~ "ww" ]]; then
-                if [ -d "$HOME/src/subg_rfspy/" ]; then
-                    echo "$HOME/src/subg_rfspy/ already exists; pulling latest"
-                    (cd $HOME/src/subg_rfspy && git fetch && git pull) || die "Couldn't pull latest subg_rfspy"
-                else
-                    echo -n "Cloning subg_rfspy: "
-                    (cd $HOME/src && git clone https://github.com/ps2/subg_rfspy) || die "Couldn't clone oref0"
-                fi
+        fi
+    elif [[ "$ttyport" =~ "mmeowlink" ]]; then # e.g. for TI USB
+        # radio_locale requires openaps 0.2.0-dev or later
+        openaps device add pump mmeowlink subg_rfspy $ttyport $serial $radio_locale || die "Can't add pump"
+        #openaps alias add wait-for-silence '! bash -c "(mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 1 | grep -q comms && echo -n Radio ok, || openaps mmtune) && echo -n \" Listening: \"; for i in $(seq 1 100); do echo -n .; mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 30 2>/dev/null | egrep -v subg | egrep No && break; done"'
+        #openaps alias add wait-for-long-silence '! bash -c "echo -n \"Listening: \"; for i in $(seq 1 200); do echo -n .; mmeowlink-any-pump-comms.py --port '$ttyport' --wait-for 45 2>/dev/null | egrep -v subg | egrep No && break; done"'
+        # TODO: check in codebase if https://github.com/ps2/subg_rfspy/tree/master/tools python scripts are still used
+        if [[ ${radio_locale,,} =~ "ww" ]]; then
+           if [ -d "$HOME/src/subg_rfspy/" ]; then
+              echo "$HOME/src/subg_rfspy/ already exists; pulling latest"
+              (cd $HOME/src/subg_rfspy && git fetch && git pull) || die "Couldn't pull latest subg_rfspy"
+            else
+              echo -n "Cloning subg_rfspy: "
+              (cd $HOME/src && git clone https://github.com/ps2/subg_rfspy) || die "Couldn't clone oref0"
             fi
-
-            # Hack to check if radio_locale has been set in pump.ini.
-            # It will remove empty line at the end of pump.ini and then append radio_locale if it's not there yet
-            grep -q radio_locale pump.ini ||  echo "$(< pump.ini)" > pump.ini ; echo "radio_locale=$radio_locale" >> pump.ini
         fi
-    else
+    else # in all other case create a pump.ini with serial and radio_locale. TODO: can we skip creating a pump.ini. I can't find references of it's being used
```

Comment:
I think that xdrip-js uses `pump.ini`, we should let them know before removing it: @efidoman @jpcunningh @thebookins

Reply:
i don't want to suggest to remove it now. As far as I can see I removed all the references to the pump.ini from the oref0 code base. I hope @efidoman @jpcunningh @thebookins can say if their software somehow uses pump.ini. I just checked Lookout, but can't find any reference: https://github.com/xdrip-js/Lookout/search?q=pump.ini&unscoped_q=pump.ini

Reply:
The only use might be for `fakemeter`, which is here: https://github.com/xdrip-js/Logger/blob/f7228485c7a66300f280ac789b5dce0252ffa279/xdrip-get-entries.sh#L245

Reply:
Created https://github.com/xdrip-js/Logger/issues/137 for that